Feat/800 fix package problems

TheHadiAhmadi opened the pull request.

mhmdtaherian commented:
@TheHadiAhmadi Please move El and flex changes to another branch. This branch should contain only fix for `tabler.css` in npm package.

pournasserian commented:
@TheHadiAhmadi what is the issue?!

TheHadiAhmadi commented:
I added flexDirection utility for El, added styles for fontSize in El
And fixed a problem in Input component (inputmask's error in SSR mode)

And moved tabler.css to npm package which I did this also in another PR #836
